src/hooks/calls/organization/useOrganization.js

```diff
@@ -22,7 +22,6 @@ export function useOrganization({ organizationName, organizationClient, options,
     },
     actions: {
       setOrganization,
-      getOrganizationClient: () => fetchOrg(),
```

Comment:
You could have a function to get the organization client used by this hook if you wanted, that might be useful. `getOrganizationClient: () => _organizationClient`

That would return the organizationClient used by the hook to make the requests.